src/main/kotlin/blackjack/card/deck/BlackJackCardDeck.kt

```diff
@@ -0,0 +1,28 @@
+package blackjack.card.deck
+
+import blackjack.card.Card
+import blackjack.card.signature.CardOrdinalSignature
+import blackjack.card.signature.CardSignaturePack
+import blackjack.card.signature.CardSymbolSignature
+
+class BlackJackCardDeck {
+    private val cardSets = fillDeck()
```

Comment:
리턴값을 생략이 가능하지만, 명시적으로 작성해주시는게 좋을거 같아요,
구현부를 보기전까지, 타입을 알기도 힘들고,
구현부가 변경되면서, 의도치 않게 타입이 변경되기도합니다!
```suggestion
    private val cardSets : ArrayDeque<Card> = fillDeck()
```